File: Java/Stacks/Stacks.java
public class Stacks {
    static final int MAX = 100;
    int arr[] = new int[MAX];
    int top;

    Stacks() {
        top = -1;
    }

    boolean isEmpty() {
        return (top < 0);
    }

    boolean push(int x) {
        if (top > (MAX - 1)) {
            System.out.println("Stack Overflow");
            return false;
        }
        arr[++top] = x;
        System.out.println(x + "Pu shed into the stack.");
        return true;
    }

    int pop() {
        if (top < 0) {
            System.out.println("Stack underflow");
            return -1;
        }

        int temp = arr[top--];
        return temp;
       

    }

    public static void main(String args[]) {
    }
}

File: cpp/Stacks/Stacks.cpp
#include <iostream>
#include <vector>vsvfs
#include <stack>
using namespace std;

const int MAX = 100;

class Stack
{
private:
    int max = 100;

public:
    int MAX = 100;
    int top;
    int a[MAX]; // Maximum size of Stack

    Stack()
    {
        top = -1;
    }

    bool push(int x);
    int pop();
    int peek();
    bool isEmpty();
};

bool Stack ::push(int x)
{
    if (top >= (MAX - 1))
    {
        cout << "Stack Overflow";
        return false;
    }
    else
    {
        int x = a[top--];
        return x;
    }
}

int main()
{

    cout << endl;
}